platform/kube/inject/testdata/auth.cert-dir.yaml.injected

```diff
@@ -64,6 +64,7 @@ spec:
         name: istio-config
       - emptyDir:
           medium: Memory
+          sizeLimit: "0"
```

Comment:
emptyDir does not seem to like this even though the API ref lists it as an option. I'm trying to determine if the Memory setting cannot be used in union with sizeLimit.